db/seeds.rb

```diff
@@ -0,0 +1,3 @@
+Api::OpenidConnect::Scope.find_or_create_by!(name: "openid")
+Api::OpenidConnect::Scope.find_or_create_by!(name: "read")
+Api::OpenidConnect::Scope.find_or_create_by!(name: "write")
```

Comment:
Not having to change our setup instructions is another point towards above point about the Scope model